src/Data/V/Linear.hs

```diff
@@ -36,17 +37,20 @@
 -- rather than linear types).
 module Data.V.Linear
   ( V,
+    next#,
+    next,
```

Comment:
These are not good names for vector operations. I guess they could be called `uncons` or something. Though I'm not sure whether there is impetus to export them at all.

Reply:
`uncons` looks good to me. They were part of the original API (named `split`). Do we have any major reason to preserve the old API?

Reply:
Not really. Not this at the very least. I probably exported this function myself… so it's my bad.